coding-standards/UI.md

```diff
@@ -0,0 +1,505 @@
+| 82  | White space                       | Design                   | White space (or negative space) is an important design element that can be used to improve readability and create a sense of balance in the UI. So always use white space effectively.                                                                                                                                                                                |
+# Fields
+
+## Names
+
+Field names or Labels are what helps the user understand which data goes where. Giving the correct & comprehensible labels is very essential for a good user interface. While most of them appear simple & straight forward, maintaining uniformity and correctness is important.  For example, Name & Full Name actually might mean the same thing while explicitly asking for First Name, Middle Name and Last Name makes it extremely clear.
+
+And when a single field has to be used, it is better to use Full Name instead of just Name. Maintaining the same label across all forms is very important. 
+
+Comments, Remarks, Notes are another set of labels are mixed up often and confuse the client. Whatever is chosen shall be maintained same across the application.
+
+One of the common mistakes noticed is to have names appear in camel notation. For ex, ExpectedTime instead of writing Expected time. This is prominently noticed in grids when the data table binding happens. This shall be avoided and names written clearly with appropriate spaces.
+
+## Alignments
+
+Fields are aligned horizontally and vertically.  In addition to alignment, the spacing between fields both horizontally and vertically is important. 
+
+**Label Vertical alignment** – all labels must be either top aligned or center aligned. Osmosys by default uses top alignment.
+
+**Label Horizontal alignment** – all labels must be either left aligned or right aligned. Osmosys by default uses left alignment.
+
+**Label / Field Vertical spacing** - This is the space between two fields in the same column. It is very important to maintain this same across all fields in the page & application.
+
+<table style="width: 562.5px; height: 157px;">
+   <tbody>
+      <tr style="height: 31px;">
+         <td style="width: 295px; height: 31px;"></td>
+         <td style="width: 265.5px; height: 31px;">Osmosys Default
+         </td>
+      </tr>
+      <tr style="height: 31px;">
+         <td style="width: 295px; height: 31px;">Label Vertical Alignment
+         </td>
+         <td style="width: 265.5px; height: 31px;">Top
+         </td>
+      </tr>
+      <tr style="height: 31px;">
+         <td style="width: 295px; height: 31px;">Label Horizontal Alignment
+         </td>
+         <td style="width: 265.5px; height: 31px;">Left
+         </td>
+      </tr>
+      <tr style="height: 31px;">
+         <td style="width: 295px; height: 31px;">Label Font Type
+         </td>
+         <td style="width: 265.5px; height: 31px;"></td>
```

Comment:
These label font type and label font size there are no values against them.

Reply:
Unsure of the values to keep, the table was converted to markdown, but kept these values blank only for now